src/__tests__/BoardSizeForm.test.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { shallow, mount } from 'enzyme';
+import BoardSizeForm from '../components/BoardSizeForm';
+
+describe('BoardSizeForm component', () => {
+  it('renders a form', () => {
+    const wrapper = shallow(
+      <BoardSizeForm
```

Comment:
So ... I'm lazy, and don't like to pass props that I'm not using. Do you like to pass props that you're not using? I didn't think so. The verbosity of this component's interface is a great opportunity to start using `defaultProps`, and passing only the props you need.